core/Session/SaveHandler/DbTable.php

```diff
@@ -23,6 +23,9 @@
  */
 class DbTable implements Zend_Session_SaveHandler_Interface
 {
+    const SESSION_DATA_MAX_LEN = 65535;
```

Comment:
Isn't the database field of type `MEDIUMTEXT` and would be able to store 16,777,215 chars?
Or is that limit coming from somewhere else?

Reply:
@sgiehl this is the column I'm looking at: https://github.com/matomo-org/matomo/blob/4.x-dev/core/Db/Schema/Mysql.php#L278 . If I'm missing something, let me know.

Reply:
fyi on demo2 and locally it is mediumtext. not sure if that was changed at some point

Reply:
@tsteur this is what i see on demo2:

```
MariaDB [74850_demo2]> show columns in `session`;
+----------+--------------+------+-----+---------+-------+
| Field    | Type         | Null | Key | Default | Extra |
+----------+--------------+------+-----+---------+-------+
| id       | varchar(191) | NO   | PRI | NULL    |       |
| modified | int(11)      | YES  |     | NULL    |       |
| lifetime | int(11)      | YES  |     | NULL    |       |
| data     | text         | YES  |     | NULL    |       |
+----------+--------------+------+-----+---------+-------+
4 rows in set (0.001 sec)

MariaDB [74850_demo2]> 
```

can you confirm? I think we're looking at two different things, but I'm not sure what.

Reply:
weird, seeing this now on demo2 the same but locally still seeing mediumtext. BTW maybe we also generally want to change the field to mediumtext?

Reply:
We could, this would be another option. I don't know if we still want to be able to show more than 30 notifications at a time (I don't personally think there's value in this since no one will read through 30 notifications let alone 100).

Reply:
Just a note: I'm also seeing `data TEXT` locally.

Reply:
Was more meaning it changing it additionally to prevent similar other issues.

Reply:
@tsteur I know, that's what I meant by "this would be another option". I'll change it and create an update to change the column. Then we won't need the warning.

I'll also keep the 30 notification limit change that's currently in the PR. If you see anything wrong w/ that let me know.